src/main/java/org/broadinstitute/dsm/analytics/GoogleAnalyticsMetricsTracker.java

```diff
@@ -0,0 +1,98 @@
+package org.broadinstitute.dsm.analytics;
+
+import com.brsanthu.googleanalytics.GoogleAnalytics;
+import com.brsanthu.googleanalytics.GoogleAnalyticsConfig;
+import com.brsanthu.googleanalytics.request.EventHit;
+import com.brsanthu.googleanalytics.request.GoogleAnalyticsResponse;
+import com.typesafe.config.Config;
+import lombok.NonNull;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class GoogleAnalyticsMetricsTracker {
+    private static final Logger logger = LoggerFactory.getLogger(GoogleAnalyticsMetricsTracker.class);
+    private static final String GA_TOKEN_PATH = "GoogleAnalytics.trackingId";
+    private static GoogleAnalytics googleAnalyticsTrackers;
+    private static volatile GoogleAnalyticsMetricsTracker instance;
+    private static Object lockGA = new Object();
+    private static Config CONFIG;
+
+    private GoogleAnalyticsMetricsTracker() {
+        initStudyMetricTracker();
+    }
+
+    public static void setConfig(@NonNull Config config){
+        CONFIG = config;
+    }
+
+    public static GoogleAnalyticsMetricsTracker getInstance() {
+        if (instance == null) {
+            synchronized (lockGA) {
+                if (instance == null) {
+                    instance = new GoogleAnalyticsMetricsTracker();
+                }
+            }
+        }
+        return instance;
+    }
+
+    private GoogleAnalytics getMetricTracker() {
+        if (googleAnalyticsTrackers == null) {
+
+            initStudyMetricTracker();
+        }
+        return googleAnalyticsTrackers;
+    }
+
+    private synchronized void initStudyMetricTracker() {
```

Comment:
if you change the code according the the comment about initializing `GoogleAnalyticsMetricsTracker` directly on field declaration, you will not need `synchronized` keyword anymore.